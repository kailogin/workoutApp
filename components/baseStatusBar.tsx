import React from "react";
import { StatusBar, View } from "react-native";
import { Colors } from "../utils/theme";

export const BaseStatusBar = () => {
  // --- RENDER ---

  return (
    <View>
      <StatusBar
        animated={true}
        backgroundColor={Colors.BLACK}
        barStyle="light-content"
        hidden={false}
      />
    </View>
  );
};
