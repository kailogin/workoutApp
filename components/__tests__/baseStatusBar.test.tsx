import "react-native";
import React from "react";
import renderer from "react-test-renderer";

import { BaseStatusBar } from "../baseStatusBar";

test(`BaseText snapshot`, () => {
  const snap = renderer.create(<BaseStatusBar />).toJSON();

  expect(snap).toMatchSnapshot();
});
