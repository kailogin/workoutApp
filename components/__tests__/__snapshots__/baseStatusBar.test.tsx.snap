// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseText snapshot 1`] = `<View />`;
